charts/minikube.md

```diff
@@ -0,0 +1,55 @@
+### Requirements
+- [minikube](https://github.com/kubernetes/minikube/#installation)
+  - [virtualbox](https://www.virtualbox.org/wiki/Downloads)
+- [helm](https://github.com/kubernetes/helm#install)
+
+The following assumes you have minikube (which requires virtualbox or other virtualization options)
+as well as helm installed (see links above). Once minikube and helm are installed, start minikube 
+with at least version 1.9 of kubernetes and perferably with 4GB of RAM (although the default value 
+of 2GB should work, we recommend 4GB or more)
+```shell
+minikube start --kubernetes-version=v1.9.0 --memory 4096
+```
+
+
+Once minikube has started, run the helm initialization. 
+#### NOTE that once you helm init, it may take a couple minutes for the Tiller services to fully deploy on your minikube
+```
+helm init
+```
+
+Clone the repo and go into the charts directory
+```shell
+git clone https://github.com/scality/Zenko.git
+cd Zenko/charts
```

Comment:
This likely won't work right now as-is...

Reply:
You're right I was thinking too far ahead. I'll add a checkout to ft/k8s-helm for when this gets merged.